buildSrc/version.properties

```diff
@@ -1,79 +1 @@
 opensearch        = 3.0.0
```

Comment:
There are some external references to the `buildSrc/version.properties` file like [here](https://github.com/opensearch-project/opensearch-build/blob/main/.github/workflows/os-increment-plugin-versions.yml#L76)